modules/services/autoraise.nix

```diff
@@ -0,0 +1,160 @@
+            };
+            delay = lib.mkOption {
+              type = nullOr int;
+              default = null;
+              description = "Raise delay, specified in units of pollMillis. Disabled if 0. A delay > 1 requires the mouse to stop for a moment before raising.";
+            };
+            focusDelay = lib.mkOption {
+              type = nullOr int;
+              default = null;
+              description = "Focus delay, specified in units of pollMillis. Disabled if 0. A delay > 1 requires the mouse to stop for a moment before focusing.";
+            };
+            warpX = lib.mkOption {
+              type = nullOr (addCheck float (x: x >= 0 && x <= 1));
+              default = null;
+              description = "A Factor between 0 and 1. Makes the mouse jump horizontally to the activated window. By default disabled.";
+            };
+            warpY = lib.mkOption {
+              type = nullOr (addCheck float (x: x >= 0 && x <= 1));
+              default = null;
+              description = "A Factor between 0 and 1. Makes the mouse jump vertically to the activated window. By default disabled.";
+            };
+            scale = lib.mkOption {
+              type = nullOr float;
+              default = null;
+              description = "Enlarge the mouse for a short period of time after warping it. The default is 2.0. To disable set it to 1.0.";
+            };
+            altTaskSwitcher = lib.mkOption {
+              type = nullOr bool;
+              default = null;
+              description = "Set to true if you use 3rd party tools to switch between applications (other than standard command-tab).";
+            };
+            ignoreSpaceChanged = lib.mkOption {
+              type = nullOr bool;
+              default = null;
+              description = "Do not immediately raise/focus after a space change. The default is false.";
+            };
+            invertIgnoreApps = lib.mkOption {
+              type = nullOr bool;
+              default = null;
+              description = "Turns the ignoreApps parameter into an includeApps parameter. The default is false.";
+            };
+            ignoreApps = lib.mkOption {
+              type = nullOr (listOf str);
+              default = null;
+              description = "Comma separated list of apps for which you would like to disable focus/raise.";
+            };
+            ignoreTitles = lib.mkOption {
+              type = nullOr (listOf str);
+              default = null;
+              description = "Comma separated list of window titles (a title can be an ICU regular expression) for which you would like to disable focus/raise.";
+            };
+            stayFocusedBundleIds = lib.mkOption {
+              type = nullOr (listOf str);
+              default = null;
+              description = "Comma separated list of app bundle identifiers that shouldn't lose focus even when hovering the mouse over another window.";
```

Comment:
Don't need to specify that the list is comma-separated, since that is abstracted away